data/json/npcs/other/TALK_NPC_APARTMENT_SURVIVOR.json

```diff
@@ -0,0 +1,71 @@
+    },
+    "speaker_effect": { "effect": { "npc_add_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } },
+    "responses": [
+      {
+        "text": "&Hold up your hands.  \"Don't worry, I'm not going to hurt you\"",
+        "topic": "TALK_NPC_APARTMENT_SURVIVOR_CALM",
+        "condition": { "not": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "Hand over your stuff!  Don't make any sudden moves, or you die!",
+        "trial": { "type": "INTIMIDATE", "difficulty": 30 },
+        "success": { "topic": "TALK_WEAPON_DROPPED", "effect": "drop_weapon", "opinion": { "trust": -4, "fear": 3 } },
+        "failure": { "topic": "TALK_DONE", "effect": "hostile" },
+        "condition": { "not": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "<greet>",
+        "topic": "TALK_NPC_APARTMENT_SURVIVOR_INTRO",
+        "condition": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      { "text": "Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_APARTMENT_SURVIVOR_CALM",
+    "dynamic_line": "Oh, you're not one of the <zombies>.  Come on in, just don't cause any trouble.",
+    "responses": [
+      { "text": "Alright, let's chat.", "topic": "TALK_NPC_APARTMENT_SURVIVOR_INTRO" },
+      { "text": "I'll be back later.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_APARTMENT_SURVIVOR_INTRO",
+    "dynamic_line": "So, what's up?",
+    "responses": [
+      { "text": "What are you doing in this deathtrap?", "topic": "TALK_NPC_APARTMENT_SURVIVOR_STORY" },
+      { "text": "Care to trade?", "topic": "TALK_NPC_APARTMENT_SURVIVOR_INTRO", "effect": "start_trade" },
+      {
+        "text": "Why don't you come with me?",
+        "trial": { "type": "PERSUADE", "difficulty": -15, "mod": [ [ "value", 2 ] ] },
+        "success": { "topic": "TALK_AGREE_FOLLOW", "effect": "follow", "opinion": { "trust": 1, "value": 1 } },
+        "failure": { "topic": "TALK_DENY_FOLLOW", "effect": "deny_follow", "opinion": { "trust": 0, "fear": 0 } }
+      },
+      {
+        "text": "Any jobs you need done?",
+        "condition": { "and": [ { "not": "has_assigned_mission" }, { "not": "has_many_assigned_missions" } ] },
+        "topic": "TALK_MISSION_LIST"
+      },
+      { "text": "I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_APARTMENT_SURVIVOR_STORY",
+    "dynamic_line": "I used to live here before <the_cataclysm>.  I was asleep when a horde of <zombies> bust through the front, and in the chaos I wound up here.  When the screaming stopped, I peaked out the door to find my neighbor stumbling about with his face half-eaten, and I haven't left since.",
```

Comment:
```suggestion
    "dynamic_line": "I used to live here before <the_cataclysm>.  I was asleep when a horde of <zombies> busted through the front, and in the chaos I wound up here.  When the screaming stopped, I peaked out the door to find my neighbor stumbling about with his face half-eaten, and I haven't left since.",
```